docs/source/notebooks/09_Hydrological_impacts_of_climate_change.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can we fix the max_temp&lt;min_temp warnings ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I added a filter to stop showing the warnings here.</p>